lib/checksum.go

```diff
@@ -0,0 +1,73 @@
+package lib
+
+import (
+	"bufio"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"golang.org/x/crypto/openpgp"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// getChecksumFromFile Extract the checksum from the signature file
+func getChecksumFromHashFile(signatureFilePath string, terraformFileName string) (string, error) {
+	readFile, err := os.Open(signatureFilePath)
+	if err != nil {
+		fmt.Println("[Error] : Could not open ", signatureFilePath)
```

Comment:
I think it makes sense to sync with https://github.com/warrensbox/terraform-switcher/pull/350 so that we don't get a mess again when #350 is merged.
Would it make sense to complete #350 first and then get back to this PR?